keras/optimizers.py

```diff
@@ -11,8 +12,28 @@
 
 
 def clip_norm(g, c, n):
-    if c > 0:
-        g = K.switch(n >= c, g * c / n, g)
+    if c <= 0:
+        return g
+    if K.backend() == 'tensorflow':
```

Comment:
I wasn't sure which block you meant so I added a comment to both..
I hope it's clear now, let me know